docs/developer-docs/latest/guides/process-manager.md

```diff
@@ -42,6 +42,12 @@ The basic usage to start an application with PM2 will be to run a command like t
 But here we are facing an issue. In your project you don't have a `.js` file to run your Strapi application.
 
 So first let's create a `server.js` file that will let you run the `pm2` command.
+::: note
+TypeScript projects require additional code in the `server.js` file to identify the correct directory. See the following TypeScript code example or the [TypeScript documentation](/developer-docs/latest/development/typescript.md) for additional details.
```

Comment:
Maybe it would be clearer to link directly the "Start Strapi x TS programmatically" part of the documentation, what do you think?

(talking about this link https://documentation-git-dev-typescript-pm2-fix-strapijs.vercel.app/developer-docs/latest/development/typescript.html#start-strapi-programmatically)

Reply:
I agree, updated.